src/Dropdown/Common/Update.elm

```diff
@@ -0,0 +1,30 @@
+module Dropdown.Common.Update exposing (update)
+
+import Dropdown.Types exposing (..)
+import Dropdown.Zipper as Zipper
+
+
+update :
+    UpdateMethods
```

Comment:
The idea here is to take the code that differs between different kinds of dropdowns (in particular, between single-select ones and multi-select ones) and put those functions into records of methods that can be passed into common code, like here. `UpdateMethods` is nearly trivial at the moment, but `ViewMethods` actually has a bunch in it,